msteams-platform/concepts/device-capabilities/location-capability.md

```diff
@@ -10,54 +10,58 @@ ms.author: surbhigupta
 
 # Integrate location capabilities
 
-You can integrate the location capabilities of native device with your Teams app.  
```

Comment:
Please change the doc structure to:
1. Overview
2. Advantages
3. Images specific to mobile and desktop with relevant text
4. Update manifest
5. Location APIs
6. Error handling
7. Code snippets
8. Code sample